agreement/security/agreement_security.xml

```diff
@@ -1,26 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <!--
   © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 -->
-
 <odoo noupdate="1">
-
-
-<record id="agreement_rule" model="ir.rule">
-    <field name="name">Agreement multi-company</field>
-    <field name="model_id" ref="model_agreement"/>
-    <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
-</record>
-
-<record id="group_use_agreement_type" model="res.groups">
-    <field name="name">Use agreement type</field>
-    <field name="category_id" ref="base.module_category_hidden"/>
-</record>
-
-<record id="group_use_agreement_template" model="res.groups">
-    <field name="name">Use agreement template</field>
-    <field name="category_id" ref="base.module_category_hidden"/>
-</record>
-
+    <record id="agreement_rule" model="ir.rule">
+        <field name="name">Agreement multi-company</field>
+        <field name="model_id" ref="model_agreement" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
This rule must be transformed and migration scripts should be added. See in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0 the line about that.